test/framework/exec/command.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
+
+import (
+	"io"
+	"io/ioutil"
+	"os/exec"
+
+	"github.com/pkg/errors"
+)
+
+// Command wraps exec.Command with specific functionality.
+// This differentiates itself from the standard library by always collecting stdout and stderr.
+// Command improves the UX of exec.Command for our specific use case.
+type Command struct {
```

Comment:
Is this type really needed? `exec.Command` is pretty light weight already and for combining stdout and stderr for `Run()` you can use `CombinedOutput` for that.

Reply:
The assumption in this comment is that I don't know how `exec.Command` works and frankly I don't appreciate that assumption. Please read the code and the godoc to understand why this type exists.

Reply:
Apologies, I see it now -- thanks!